Templates/CSharp/Requests/EntityCollectionRequestBuilder.cs.tt

```diff
@@ -5,6 +5,12 @@
 
 var prop = host.CurrentType.AsOdcmProperty();
 
+    var collectionRequestBuilderConstructor = this.GetCollectionRequestBuilderConstructor(prop);
```

Comment:
I made these changes in an attempt to limit junk whitespace changes to the generated code files. I now know that resistance is futile.